src/providers/wcs/qgswcscapabilities.h

```diff
@@ -34,6 +34,13 @@
 class QNetworkAccessManager;
 class QNetworkReply;
 
+//! Metadata Link Property structure
+struct QgsWcsMetadataLinkProperty
+{
+  QString metadataType;
```

Comment:
Can you document the struct members too?

And in general, add something new to a doc string (additional to repeating the name of the struct/class/method/...). The CoverageSummary example below is already quite bad.